redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/ZiplineTreehouseUi.kt

```diff
@@ -33,5 +33,15 @@ public interface ZiplineTreehouseUi : ZiplineService, EventSink {
     uiConfigurations: StateFlow<UiConfiguration>,
     stateSnapshot: StateSnapshot?,
   )
+
+  @Deprecated(
+    "delete after 2023-07-23",
```

Comment:
How long should we support backwards-incompatible Redwood internals before 1.0 ?

Reply:
Once 0.5 drops it has to be all the way to 1.0.

But since 0.5 hasn't dropped, why are we bothering to do this at all?

Reply:
We’ve got some employees using it already!

Reply:
Then they are doing so without a guarantee of stability and breaking them is perfectly within our right.